### DISCUSSION TITLE
Empty state in tables

### DISCUSSION THREAD

**Discussion by KateSniegocka** [twilio-labs/paste#2278] [Office hours] [answered]
**Background:** We are working on the project to move Console to Paste for Flex Admin experience. 

**Scenario:** One of the use cases recently discovered is an empty state in the table after using filtering. 
If the user will filter through the available channels and there are no results to show in the table as a result of this action. 

**Question:** How do we best address that scenario in Paste? 
Some images below to help to explain and visualise: 

**Image 1  No filters - results available**
![im 1](https://user-images.githubusercontent.com/97667802/157313657-6f6cc853-e0ae-41b1-b3d1-a847683c2ec8.png)

**Image 2  Filtered - no results = empty state**
![im 2](https://user-images.githubusercontent.com/97667802/157314411-eb983e77-088e-494e-9d03-173c5730e6e9.png)

### Proposed solutions/options:

**1. Dashes**
![im 3](https://user-images.githubusercontent.com/97667802/157315687-bbd6f654-8f08-4c10-aa0a-8745de79cdbf.png)

**2. Text**
![im 4](https://user-images.githubusercontent.com/97667802/157315824-e65bb555-dec3-4b13-aba7-ed437e8f5826.png)

**3. Icon + Text**
![im 5](https://user-images.githubusercontent.com/97667802/157315880-7981abec-2b15-46c3-bc98-1a1f428e4ec2.png)

**4. Illustration + Text**
![im 6](https://user-images.githubusercontent.com/97667802/157315961-6c705d65-1f3a-4e32-985c-6fb7e7fb41f6.png)

**Comment by shleewhite** [comment, marked as answer]
Thank you for coming to office hours! Just to recap, we discussed using:
* [the Create pattern](https://paste.twilio.design/patterns/create/#standard) for the add new address button
* [the Empty state pattern](https://paste.twilio.design/patterns/empty-state/#examples-of-empty-state-use-cases) for when the filters return no results
* if there is a case where there are no addresses yet and there are no filters applied, you could use the no data empty state illustration.

Also, here is the File where Loreina made some potential designs: https://www.figma.com/file/SpeLpBoqN2vZv8KWcSn0J1/Paste-Feedback?node-id=1214%3A113590
